httptesting/http.go

```diff
@@ -110,6 +113,10 @@ func AssertJSONCall(c *gc.C, p JSONCallParams) {
 		return
 	}
 	AssertJSONResponse(c, rec, p.ExpectStatus, p.ExpectBody)
+
+	for key, value := range p.ExpectHeader {
+		c.Assert(rec.HeaderMap[key], gc.DeepEquals, value)
```

Comment:
I think this should probably be:

     c.Assert(rec.HeaderMap[key], gc.DeepEquals, value, gc.Commentf("header %q", key))

so that we have some idea of what header the problem is with when the test fails.